intel_owl/settings/__init__.py

```diff
@@ -16,6 +16,8 @@
     "django.contrib.messages",
     "django.contrib.staticfiles",
     "django.contrib.postgres",
+    # sites
```

Comment:
could you specify here as a comment that this package is required for the `allauth` installation? otherwise people in future could wonder why it is here.